pkg/apply/merge.go

```diff
@@ -200,6 +204,26 @@ func mergeLabels(current, updated *uns.Unstructured) {
 	}
 }
 
+func mergeConfigMapForUpdate(current, updated *uns.Unstructured) error {
+	s1, ok, err := uns.NestedStringMap(current.Object, "data")
```

Comment:
if this is a ConfigMap function please add
```
gvk := updated.GroupVersionKind()
	if gvk.Group == "" && gvk.Kind == "ConfigMap" {
```

Reply:
good idea!